module/app/app.go

```diff
@@ -103,6 +106,8 @@ import (
 	_ "github.com/cosmos/cosmos-sdk/client/docs/statik"
 
 	gravityparams "github.com/Gravity-Bridge/Gravity-Bridge/module/app/params"
+	"github.com/Gravity-Bridge/Gravity-Bridge/module/app/upgrades"
+	v2 "github.com/Gravity-Bridge/Gravity-Bridge/module/app/upgrades/v2"
```

Comment:
should we rename all the 'v2' references to 'mercury'? I could go either way on this.

Reply:
Going with v2 in the code for simplicity later on, mercury in the UI